### ISSUE TITLE
How to change the price of the shotgun on the terminal

### ISSUE THREAD

**Issue by StupidClownPT** [radsi/LC-Shotgun#2]
Hi there,
Im really enjoying this mod and having fun with my friends but i wanted to make it so you actually have to "play" to earn the shotgun
And i wanted some help in knowing how to change its prince on the terminal

Thanks in Advance

**Comment by JakeBiggs** [comment]
How do you even buy it on the terminal? it seems to buy a shovel when i type shotgun

**Comment by StupidClownPT** [comment]
Do you have the mod installed correctly?
Does it appear on the terminal?
If it does you might be typing it wrong or something idk it works for me

**Comment by Saskia51** [comment]
> Le mod est-il correctement installé ? Apparaît-il sur le terminal ? Si c'est le cas, vous avez peut-être mal tapé ou quelque chose comme ça, ça marche pour moi

Bonjour , j'ai le même problème je ne le voit pas sur le terminal et j'achète une pelle à la place , pourtant je pense que je l'ai bien installé mais impossible de pouvoir l'avoir .

**Comment by JakeBiggs** [comment]
Yeah my bad i was using a deprecated version of BepInEx. love the mod btw had a good look through the code too. cool shit On 6 Dec 2023, at 20:32, StupidClownPT ***@***.***> wrote:﻿
Do you have the mod installed correctly?
Does it appear on the terminal?
If it does you might be typing it wrong or something idk it works for me
